.tools/test/stacks/deploy.py

```diff
@@ -31,9 +30,12 @@ def run_shell_command(command, env_vars=None):
     try:
         output = subprocess.check_output(command, stderr=subprocess.STDOUT, env=env)
         print(f"Command output: {output.decode()}")
-    except Exception as e:
+    except subprocess.CalledProcessError as e:
         print(f"Error executing command: {e.output.decode()}")
         raise
+    except Exception as e:
+        print(f"Exception executing command: {e!r}")
```

Comment:
Never seen `e!r` before. Interesting.

Reply:
Forces the __repr__, rather than the __str__, though many __str__ fall back to __repr__.